File: Programming Basics with Python/T02 - First Steps in Coding/exercise/06. Repainting.py
nylon = int(input())
paint = int(input())
razr = int(input())
hours_workers = int(input())

price_nylon = (nylon + 2) * 1.50
price_paint = (paint * 1.10) * 14.50
price_razr = razr * 5

sum_materials = price_nylon + price_paint + price_razr + 0.40
sum_workers = (sum_materials * 0.30) * hours_workers

total_sum = sum_materials + sum_workers

print(total_sum)

File: Programming Basics with JS/T02 - First Steps in Coding/exercise/06. Repainting.js
function repainting(input) {
    let nylon = Number(input[0]);
    let paint = Number(input[1]);
    let thinner = Number(input[2]);
    let hours = Number(input[3]);
    let priceNylon = (nylon + 2) * 1.5;
    let pricePaint = (paint * 1.10) * 14.5;
    let priceThinner = thinner * 5;
    let totalStuffPrice = priceNylon + pricePaint + priceThinner + 0.40;
    let workPricePerHour = totalStuffPrice * 0.30;
    let totalWorkPrice = workPricePerHour * hours;
    let totalPrice = totalStuffPrice + totalWorkPrice;

    console.log(totalPrice);
}
// repainting(["10", "11", "4", "8"]);
repainting(["5", "10", "10", "1"]);